app/src/main/java/ch/sdp/vibester/database/DataGetter.kt

```diff
@@ -184,6 +185,10 @@ class DataGetter @Inject constructor() {
         dbRoomRef.child(partyRoom.getRoomID()).child("emailList").setValue(partyRoom.getEmailList())
     }
 
+    fun getCurrentUser(): FirebaseUser? {
+        return authenticator.getCurrUser()
```

Comment:
why is it in dataFetter? We need to think of a separation between firebaseUser and Datagetter